.eslintrc

```diff
@@ -1,7 +1,12 @@
 {
-  "extends": ["standard", "standard-react"],
+  "extends": [
+    "standard", 
+    "standard-react", 
+    "plugin:react/recommended"
+  ],
   "rules": {
-    "space-before-function-paren": "off"
+    "space-before-function-paren": "off",
+    "react/prop-types": "off"

```

Comment:
Why we need to turn prop-types off?

Reply:
algumas propriedades como por exemplo 
`this.props.history...`
o eslint nao reconhece e fica dando erro/aviso o tempo todo e nao é necessario.